src/SFML/Audio/InputSoundFile.cpp

```diff
@@ -32,6 +32,8 @@
 #include <SFML/System/FileInputStream.hpp>
 #include <SFML/System/MemoryInputStream.hpp>
 #include <SFML/System/Err.hpp>
+#include <iostream>
```

Comment:
Where is `<iostream>` used?

Reply:
There's a usage of `std::endl` on line 156.